.github/workflows/ci.yaml

```
@@ -5,14 +5,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
         include:
           - python-version: 3.6
             toxenv: py36
           - python-version: 3.7
             toxenv: py37
           - python-version: 3.8
             toxenv: py38
+          - python-version: 3.9
+            toxenv: py39
+          - python-version: 3.10
```

yaml converts 3.10 to a number, unless you add the quotes
```suggestion
          - python-version: "3.10"
```

---

🤦 YAML is the most annoying language (except for all the other ones). Fixed in 900b98dc71aac978de954b9de4a0801c13029cad.